README.md

```diff
@@ -1,6 +1,6 @@
 # GLOMAP: Global Structure-from-Motion Revisited
```

Comment:
Shall we refer to the getting_started doc in the README?

Reply:
good point!